composer.json

```diff
@@ -23,10 +23,10 @@
         }
     },
     "require": {
-        "php": ">=7",
-        "amphp/parallel": "^1.1",
-        "amphp/amp": "^2.0.3",
-        "opis/closure": "^3.0.7"
+        "php": ">=7.4",
+        "amphp/parallel": "^1.4.1",
+        "amphp/amp": "^2.6.1",
```

Comment:
Any reason to bump these?

Reply:
Think that might have been accidental 🤔 I'll revert.